dags/common/cleanup.py

```diff
@@ -18,7 +20,37 @@ def convert_html_subsripts_to_latex(input):
 def clean_collaboration(input):
     return clean_whitespace_characters(input.replace("for the", ""))
 
+
 def remove_specific_tags(value, tags=None, attributes=None):
     tags = tags or []
     attributes = attributes or []
     return clean(value, tags=tags, attributes=attributes, strip=True)
+
+
+class RemoveUnneccessaryTagsFilter(Filter):
+    """Filter removes content of label, but not the tag itself!"""
```

Comment:
there is a python guide [PEP257](https://peps.python.org/pep-0257/) how to write python doc strings 😁  Please have a look. In general, the function/class name should be enough to understand what it does and we usually omit unnecessary docs/comments 😃